modules/rds/main.tf

```diff
@@ -0,0 +1,72 @@
+resource "aws_security_group" "this" {
+  name_prefix = "${var.name}-rds"
+  vpc_id      = var.vpc_id
+}
+
+resource "aws_security_group_rule" "cluster-rules" {
+  from_port                = module.db.this_db_instance_port
+  protocol                 = "tcp"
+  security_group_id        = aws_security_group.this.id
+  to_port                  = module.db.this_db_instance_port
+  type                     = "ingress"
+  cidr_blocks              = var.cidr_blocks
+}
+
+module "db" {
```

Comment:
Should we call the module something with "postgres" incase we one day use another rds db type for whatever reason?

Reply:
Sure thing, I have also renamed the modules/rds to modules/postgres to make it more explicit that it's a postgres database you are deploying